bicycleparameters/plot.py

```diff
@@ -95,3 +95,23 @@ def plot_eigenvalues(bikes, speeds, colors=None, linestyles=None,
         plt.show()
 
     return fig
+
+
+from numpy import pi, sin, cos
+# import plotly.graph_objects as go
+
+def ell(x_center, y_center, ax1, ax2, a, b, N = 100):
```

Comment:
```suggestion
def generate_ellipse_plot_data(x_center, y_center, ax1, ax2, a, b, N = 100):
```

functions are best named with verbs and should give more an idea of what they do. Here is a possible example of a better function name.